sumOfOthers.js

```diff
@@ -0,0 +1,13 @@
+sumOfOthers = function(array) {
+    let result = new Array(array.length).fill(0);
```

Comment:
It is not necessary to fill an array, you could just create an empty array `let result = [];`